src/app/hooks/useToggle/README.md

```diff
@@ -0,0 +1,20 @@
+# useToggle Hook
+
+## Feature Toggles
+
+Feature toggles are designed to be global (i.e not service specific) and are dependant on the application environment.
+Feature toggles can be found in `src/app/lib/config/toggles`
+
+### Using the useToggle Hook
+
+```
+import useToggle from './useToggle';
+
+const Component = () => {
+    const { enabled } = useToggle('foo')
+
+    return enabled ? <h1>toggle enabled</h1> : <p>toggle disabled</p>
+}
+
+export default Component;
+```
```

Comment:
I think in this README, we could also add how we can override toggle provider, i.e in storybook I noticed, we can ovverride the toggles provider like this:
```
const defaultToggles = {
  mediaPlayer: {
    enabled: true,
  },
};

<ToggleContextProvider
  value={{ toggleState: defaultToggles }}
  service="pidgin"
  origin="https://www.test.bbc.com"
>
```

Reply:
Thanks for updating the readme 💯